test-frontend.sh

```diff
@@ -21,7 +21,7 @@ if git status --short | grep 'yarn.lock' > /dev/null; then
 fi
 
 yarn i18n
-GIT_STATUS="$( git status --short --untracked-files -- ./locales )"
+GIT_STATUS=[$(git status --short --untracked-files -- public/locales) || $(git status --short --untracked-files -- packages/**/locales)]
```

Comment:
I believe this can be one command.

```suggestion
GIT_STATUS=[$(git status --short --untracked-files -- public/locales packages/**/locales)]
```

Reply:
All set!